mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/SendConfirmationScreenV2.tsx

```diff
@@ -0,0 +1,305 @@
+import { ThemedActivityIndicatorV2, ThemedIcon, ThemedScrollViewV2, ThemedTextV2, ThemedView, ThemedViewV2 } from '@components/themed'
+import { SummaryTitleV2 } from '@components/SummaryTitleV2'
+import { NumberRowV2 } from '@components/NumberRowV2'
+import { SubmitButtonGroupV2 } from '@components/SubmitButtonGroupV2'
+import { PortfolioParamList } from '../PortfolioNavigator'
+
+type Props = StackScreenProps<PortfolioParamList, 'SendConfirmationScreen'>
+
+export function SendConfirmationScreenV2 ({ route }: Props): JSX.Element {
+  const { address } = useWalletContext()
+  const addressLabel = useAddressLabel(address)
+  const network = useNetworkContext()
+  const {
+    token,
+    destination,
+    amount,
+    amountInUsd,
+    fee,
+    conversion
+  } = route.params
+  const logger = useLogger()
+  const hasPendingJob = useSelector((state: RootState) => hasTxQueued(state.transactionQueue))
+  const hasPendingBroadcastJob = useSelector((state: RootState) => hasBroadcastQueued(state.ocean))
+  const dispatch = useAppDispatch()
+  const [isSubmitting, setIsSubmitting] = useState(false)
+  const navigation = useNavigation<NavigationProp<PortfolioParamList>>()
+  const [isOnPage, setIsOnPage] = useState<boolean>(true)
+  const [isAcknowledge, setIsAcknowledge] = useState(false)
+
+  useEffect(() => {
+    setIsOnPage(true)
+    return () => {
+      setIsOnPage(false)
+    }
+  }, [])
+
+  async function onSubmit (): Promise<void> {
+    if (hasPendingJob || hasPendingBroadcastJob) {
+      return
+    }
+    setIsSubmitting(true)
+    await send({
+      address: destination,
+      token,
+      amount,
+      networkName: network.networkName
+    }, dispatch, () => {
+      onTransactionBroadcast(isOnPage, navigation.dispatch)
+    }, logger)
+    setIsSubmitting(false)
+  }
+
+  function onCancel (): void {
+    if (!isSubmitting) {
+      navigation.navigate({
+        name: 'Send',
+        params: {
+          token
+        },
+        merge: true
+      })
+    }
+  }
+
+  return (
+    <ThemedScrollViewV2 style={tailwind('pb-4')}>
+      <ThemedViewV2 style={tailwind('flex-col px-5 py-8')}>
+        <SummaryTitleV2
+          amount={amount}
+          title={translate('screens/SendConfirmationScreen', 'You are sending')}
+          testID='text_send_amount'
+          iconA={token.displaySymbol}
+          fromAddress={address}
+          fromAddressLabel={addressLabel}
+          toAddress={destination}
+        />
+
+        {conversion?.isConversionRequired === true &&
+          <ThemedView
+            style={tailwind('border-t-0.5 pt-5 mt-8 mb-2')}
+            light={tailwind('bg-transparent border-mono-light-v2-300')}
+            dark={tailwind('bg-transparent border-mono-dark-v2-300')}
+          >
+            <NumberRowV2
+              containerStyle={{
+                style: tailwind('flex-row items-start w-full bg-transparent '),
+                light: tailwind('bg-transparent'),
```

Comment:
Is this redundant already?

Reply:
Updated! updated `NumberRowV2` component to remove redundancy